packages/react/src/drafts/SelectPanel2/SelectPanel.tsx

```diff
@@ -306,8 +312,8 @@ const Panel: React.FC<SelectPanelProps> = ({
               }}
             >
               <Box
-                as="form"
-                method="dialog"
+                as={isInsideForm ? 'div' : 'form'}
```

Comment:
Is it possible to write a test for this to ensure we're not getting the error again? Or is it not detectable through testing?